bin/check-znode.rb

```diff
@@ -49,14 +50,26 @@ class CheckZnode < Sensu::Plugin::Check::CLI
          short: '-v',
          long: '--check_value REGEX'
 
+  option :check_child,
+         description: 'Optionally check sub-nodes against a regex',
+         short: '-c',
+         long: '--check_child REGEX'
+  
   def znode_status
     zk = Zookeeper.new(config[:server])
     znode = zk.get(path: config[:znode])
+    children = config[:check_child] ? "#{zk.get_children(path: config[:znode])[:children]}" : null
```

Comment:
rather than string interpolation use `.to_s` to cast it to a string:
```suggestion
    children = config[:check_child] ? zk.get_children(path: config[:znode])[:children].to_s : null
```

Reply:
Just to make sure it's `null` and not `nil`